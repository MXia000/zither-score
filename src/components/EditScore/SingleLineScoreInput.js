import React from "react";
import SingleScoreInput from "./SingleScoreInput";

// Map through singleInput for one line score input
const SingleLineScoreInput = props => {
  const nodeLength = [0, 1, 2, 3];

  if(props.score) {
    return (
      <SingleScoreInput 
        lineLength={props.lineLength} 
        nodeLength={nodeLength} 
        score={props.score}
      />
    );
  }
};

export default SingleLineScoreInput;

