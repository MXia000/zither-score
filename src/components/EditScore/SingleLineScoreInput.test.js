import React from 'react';
import { shallow } from 'enzyme';
import SingleLineScoreInput from './SingleLineScoreInput';

describe("SLSI", () => {
    it('render slsi without crashing', () => {  
        const SLSIComp = shallow(<div><SingleLineScoreInput /></div>);
        expect(SLSIComp).toMatchSnapshot();
    });
});