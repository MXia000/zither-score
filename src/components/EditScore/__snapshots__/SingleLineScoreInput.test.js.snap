// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SLSI render slsi without crashing 1`] = `ShallowWrapper {}`;
